farcasterd.toml

```diff
@@ -37,6 +37,23 @@ bitcoin_cookie_path = "~/.bitcoin/testnet3/.cookie"
 # the wallet should have spendable funds
 monero_rpc_wallet = "http://localhost:38084"
 
+# Swap parameter for the Bitcoin blockchain
+[swap.bitcoin]
+# Avoid broadcasting a transaction if a race can happen in # blocks
+#
+# E.g. if the safety margin is set to 3 and the the cancel transaction becomes
+# broadcastable in 4 blocks we are safe and we do publish buy, if it becomes
+# broadcastable in 3 blocks or less we do not publish the buy as the safety
+# margin is not respected.
+safety_margin = 3
+# Number of confirmations required to consider a transaction final
+finality = 1
```

Comment:
nit: for consistency, I'd either
1. `safety_margin` → `safety` or
2. `finality` → `finality_thr`/`finality_threshold` for consistency.

Reply:
What I initially disliked with `threshold` is we can put it everywhere. For now the variant I like the most is renaming `safety` and keeping `finality`. IMO the comments are very important, good descriptions are also key for a good config file.